themes/university.typ

```diff
@@ -0,0 +1,263 @@
+// University theme
+//
+// By Pol Dellaiera - https://github.com/drupol
+//
+// Please feel free to improve this theme
+// by submitting a PR in https://github.com/andreasKroepelin/typst-slides
+//
+#let university-theme(
+    university-name: "University",
+    color-a: none,
+    color-b: none,
+    color-c: none,
```

Comment:
Your choices of colors are nice, they should be the default!

Reply:
fixed.